web_assets_warmup/hooks.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Camptocamp SA
+# Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+import os
+
+import psycopg2
+
+from odoo import fields, registry
+
+logger = logging.getLogger(__name__)
+
+
+def active_cron_assets():
+    """Plan the next execution of the cron responsible to generate assets."""
+    if os.environ.get("RUNNING_ENV") == "dev":
+        return
+    dbname = os.environ.get("DB_NAME")
```

Comment:
`DB_NAME` is not a core odoo var. The db name should be take from config.
Not blocking, you can fix this later as this one is waiting since quite some time.